drafts/chinese-text/pods/encode.yml

```
@@ -0,0 +1,5 @@
+!TransformerTorchEncoder
+with:
+  pooling_strategy: auto
+  pretrained_model_name_or_path: bert-base-chinese
+  max_length: 96 default
```

@alexcg1  seems this line should be 
```suggestion
  max_length: 96
```